apps/molgenis-components/src/components/forms/ButtonAction.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <button type="button" class="btn btn-primary" @click="onClick">
+  <button type="button" class="btn btn-primary" @click.prevent="onClick">
```

Comment:
why add a prevent ?

Reply:
because the button can wrap other actions; I notice that this had as consequence that events where fired twice. (resulting in the problem that the modal doesn't close because immediately reopend). Therefore the prevent is probably a good thing we should generally do unless we want that behavior.